src/components/Auth/AuthContainer.jsx

```diff
@@ -39,6 +40,11 @@ class AuthContainer extends React.Component {
     userData.email = data.email;
     userData.name = data.name;
 
+    if (process.env.NODE_ENV) {
```

Comment:
Won't this trigger in production?

Reply:
yes that should have been `=== 'development'` fixing now